src/cpp/src/continuous_batching_pipeline.cpp

```diff
@@ -78,7 +79,7 @@ ContinuousBatchingPipeline::ContinuousBatchingPipeline( const std::filesystem::p
         auto main_model_descr = ov::genai::ModelDesc(model, tokenizer, device, properties_without_draft_model, scheduler_config, generation_config);
         m_impl = std::make_shared<SpeculativeDecodingImpl>(main_model_descr, draft_model_desr);
     } else if (std::filesystem::exists(directory / "openvino_text_embeddings_model.xml") ) {
-        auto inputs_embedder = std::make_shared<InputsEmbedder>(directory, device, properties);
+        auto inputs_embedder = std::make_shared<InputsEmbedder>(directory, device, inputs_embedder_properties.empty() ? properties : inputs_embedder_properties);
```

Comment:
1. That's a strange behavior. It's like merging `inputs_embedder_properties` and `properties` except it doesn't merge.
2. `Tokenizer` only gets its properties: `auto tokenizer = ov::genai::Tokenizer(directory, tokenizer_properties);` without falling back to `properties`. Align them.